engine/combat/utility/sos_consideration.py

```diff
@@ -0,0 +1,59 @@
+from control import sos_ctrl
+from engine.combat.appraisals.basic_attack import BasicAttack
+from engine.combat.utility.core.action import Action
+from engine.combat.utility.core.consideration import Consideration
+from memory.combat_manager import CombatCharacter
+
+
+class SoSConsideration(Consideration):
+    # Generates a list of appraisals for a character.
+    def generate_appraisals(self):
+        return self._default_appraisals() + self._character_appraisals()
+
+    # if the selected character is NONE or we are on the selected character, considered valid'
+    # and do nothing else here.
+    def valid(self, selected_character, action):
+        return selected_character is CombatCharacter.NONE or self.on_selected_character(
+            selected_character, action
+        )
+
+    def on_selected_character(self, selected_character, action):
+        return selected_character is action.consideration.actor.character
+
+    # Generates default appraisals generic to every consideration
+    # TODO: Add items?
+    def _default_appraisals(self):
+        basic_attack = BasicAttack()
+        # TODO: Dont use this as the value.
+        basic_attack.value = self.actor.physical_attack
```

Comment:
So if I get it correctly, right now the "AI" selects whichever available character that has the highest attack stat?

Reply:
Yes! For now...